### ISSUE TITLE
Column does not exist when using multiple levels of inheritance

### ISSUE THREAD

**Issue by bodgerbarnett** [ericls/django-hashids#25]
This library does exactly what I need so thanks!

However, it doesn't seem to work when using multiple levels of inheritance:-

```
class A(models.Model):
    hashid = HashidsField(real_field_name="id")

    class Meta:
        abstract = True


class B(A):
    pass


class C(B):
    pass

Python 3.9.9 (main, Feb 26 2022, 15:01:10) 
Type 'copyright', 'credits' or 'license' for more information
IPython 7.20.0 -- An enhanced Interactive Python. Type '?' for help.

In [1]: c = C.objects.create()

In [2]: C.objects.get(hashid=c.hashid)
---------------------------------------------------------------------------
UndefinedColumn                           Traceback (most recent call last)
.../django/db/backends/utils.py in _execute(self, sql, params, *ignored_wrapper_args)
     83             else:
---> 84                 return self.cursor.execute(sql, params)
     85 

UndefinedColumn: column c.id does not exist
LINE 1: ...c"."b_ptr_id" = "b"."id") WHERE "_c...
                                                             ^
HINT:  Perhaps you meant to reference the column "b.id".
```

If I only use 2 levels, it works - but I need the 3.

Any ideas?

**Comment by ericls** [comment]
Sorry about the delay, somehow missed the ticket. I'll take a look after work today. 
Might be related to #19 . 

Good to know the detail that this happens with more than 2 levels of inheritance.

**Comment by ericls** [comment]
Should be fixed in version 0.6.0(11e25d02779d41e16c72dea0a7c58e1261b2b41f)

**Comment by bodgerbarnett** [comment]
Great! Thanks so much. Any idea when 0.6.0 will be released?

**Comment by ericls** [comment]
Hi @bodgerbarnett , it should be already released now. https://pypi.org/project/django-hashids/#history

**Comment by ericls** [comment]
@bodgerbarnett , sorry that didn't fix the issue. I'll create a new PR for that.

**Comment by ericls** [comment]
hi @bodgerbarnett , it should be fixed in version 0.6.5, released just now.

**Comment by bodgerbarnett** [comment]
Superb! Working fine now. Thanks so much for your efforts!
